### DISCUSSION TITLE
Xiaomi 14 上经常崩溃

### DISCUSSION THREAD

**Discussion by tkinwa** [osfans/android-rime#1527] [General | 常规讨论]
怎么上传日志![Screenshot_2024-12-09-10-10-20-828_com.osfans.trime.jpg](https://github.com/user-attachments/assets/b27cdac4-b9ec-4f84-9e64-2e87124e9b6d)

**Comment by WhiredPlanck** [comment]
点击下方的“导出”将日志文件保存，然后将文件上传即可。
